azure/cosmos/query_iterator.py

```diff
@@ -0,0 +1,19 @@
+import collections.abc
+
+
+class QueryResultIteratable(collections.abc.Iterator):
```

Comment:
`QueryResultIteratable` > `QueryResultIterable`

Reply:
Changing to iterator